.travis.yml

```diff
@@ -1,2 +1,8 @@
 language: node_js
-node_js: "node"
\ No newline at end of file
+node_js:
+  - "8"
+  - "9"
```

Comment:
I don’t think we need to include 9 in CI. It’s not a maintained release: https://nodejs.org/en/about/releases/